app/helpers/checkin_helper.rb

```diff
@@ -0,0 +1,2 @@
+module CheckinHelper
+end
```

Comment:
Maybe remove this till it is needed.

Reply:
ah i just took this from the generator duley noted